src/promptflow/promptflow/_utils/capability_list.json

```diff
@@ -0,0 +1,12 @@
+[
+    {
+        "name": "Activate",
+        "description": "Bypass node execution when the node does not meet activate condition.",
+        "components": [
+            "runtime",
+            "executor",
+            "pfs"
+        ],
+        "state": "Ready"
```

Comment:
I think it is better to have an enum as state.

Reply:
Added.